### ISSUE TITLE
Traceur ES6 unit tests on static Angular Javascript code?

### ISSUE THREAD

**Issue by bradgreens** [karma-runner/karma-traceur-preprocessor#2]
We had the idea that it would be great to write our unit tests in ES6 and compile it to JS with Traceur. However, this is proving to be difficult to configure correctly, even with some working examples out there.

Here are my current configurations where we were previously running Coffeescript unit tests on a Rails stack, and want to replace Coffeescript with ES6.
https://gist.github.com/bradgreens/cfbee11a48db2632b08c

I worked off of these examples:
https://github.com/tpodom/karma-traceur-test
https://github.com/karma-runner/karma-traceur-preprocessor#configuration

The `test-main.js` pattern is a common theme and I'm not sure how to work with it.

In my example, Chrome launches and throws the following error:
```
Chrome 35.0.1916 (Mac OS X 10.9.2) ERROR
  Uncaught object
  at /Users/bgreenwald/Sites/playbook-web-client/node_modules/requirejs/require.js:141
```

Not so easy to debug. However, when I render the es6 script into the DOM it shows that may code has been magically wrapped inside a requireJs `define` block and I have no straightforward way to execute it.

TL;DR - I'd like to write unit tests in ES6 and run them against staticJs files which are already written to the DOM. The problem is my test - https://gist.github.com/bradgreens/cfbee11a48db2632b08c#file-services-application-spec-es6-L2 - becomes trapped in a RequireJs definition.

**Comment by jonricaurte** [comment]
@bradgreens I know this is from a while back, but babel has something like https://babeljs.io/docs/usage/modules/#ignore where you can have your tests compile down to normal javascript so they will be run right away.

**Comment by bradgreens** [comment]
Thanks for the ping. We migrated to babeljs a while back.
